configurations/dadaab_pipeline_config.json

```diff
@@ -0,0 +1,81 @@
+    {"RapidProKey": "Rqa_S01_E02_Dadaab (Run ID) - keep_II_s01_e02_dadaab_activation", "PipelineKey": "rqa_s01e02_run_id"},
+    {"RapidProKey": "Rqa_S01_E03_Dadaab (Run ID) - keep_II_s01_e03_dadaab_activation", "PipelineKey": "rqa_s01e03_run_id"},
+    {"RapidProKey": "Rqa_S01_E04_Dadaab (Run ID) - keep_II_s01_e04_dadaab_activation", "PipelineKey": "rqa_s01e04_run_id"},
+    {"RapidProKey": "Rqa_S01_E05_Dadaab (Run ID) - keep_II_s01_e05_dadaab_activation", "PipelineKey": "rqa_s01e05_run_id"},
+    {"RapidProKey": "Rqa_S01_E06_Dadaab (Run ID) - keep_II_s01_e06_dadaab_activation", "PipelineKey": "rqa_s01e06_run_id"},
+    {"RapidProKey": "Rqa_S01_E07_Dadaab (Run ID) - keep_II_s01_e07_dadaab_activation", "PipelineKey": "rqa_s01e07_run_id"},
+
+    {"RapidProKey": "Rqa_S01_E01_Dadaab (Time) - keep_II_s01_e01_dadaab_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E02_Dadaab (Time) - keep_II_s01_e02_dadaab_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E03_Dadaab (Time) - keep_II_s01_e03_dadaab_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E04_Dadaab (Time) - keep_II_s01_e04_dadaab_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E05_Dadaab (Time) - keep_II_s01_e05_dadaab_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E06_Dadaab (Time) - keep_II_s01_e06_dadaab_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "Rqa_S01_E07_Dadaab (Time) - keep_II_s01_e07_dadaab_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Gender (Text) - keep_II_dadaab_demogs", "PipelineKey": "gender_raw"},
+    {"RapidProKey": "Age (Text) - keep_II_dadaab_demogs", "PipelineKey": "age_raw"},
+    {"RapidProKey": "Location (Text) - keep_II_dadaab_demogs", "PipelineKey": "location_raw"},
+    {"RapidProKey": "Nationality (Text) - keep_II_dadaab_demogs", "PipelineKey": "nationality_raw"},
+    {"RapidProKey": "Household_Language (Text) - keep_II_dadaab_demogs", "PipelineKey": "household_language_raw"},
+
+    {"RapidProKey": "Gender (Time) - keep_II_dadaab_demogs", "PipelineKey": "gender_time"},
+    {"RapidProKey": "Age (Time) - keep_II_dadaab_demogs", "PipelineKey": "age_time"},
+    {"RapidProKey": "Location (Time) - keep_II_dadaab_demogs", "PipelineKey": "location_time"},
+    {"RapidProKey": "Nationality (Time) - keep_II_dadaab_demogs", "PipelineKey": "nationality_time"},
+    {"RapidProKey": "Household_Language (Time) - keep_II_dadaab_demogs", "PipelineKey": "household_language_time"}
+  ],
+  "PipelineName": "dadaab_pipeline",
+  "ProjectStartDate": "2019-01-01T00:00:00+03:00",
+  "ProjectEndDate": "2100-01-01T00:00:00+03:00",
+  "FilterTestMessages": false,
+  "MoveWSMessages": false,
+  "DriveUpload": {
+    "DriveCredentialsFileURL": "gs://avf-credentials/pipeline-runner-service-acct-avf-data-core-64cc71459fe7.json",
+    "ProductionUploadPath": "keep_II_analysis_outputs/dadaab/keep_II_s01_dadaab_production.csv",
```

Comment:
wusc_keep_ii

Reply:
or WUSC_KEEP_II

Reply:
updated to wusc_keep_ii everywhere except "TableName", Memory and backup folders which are now WUSC_KEEP_II